config/crd/bases/k6.io_k6s.yaml

```diff
@@ -988,13 +988,35 @@ spec:
                     type: object
                   serviceAccountName:
                     type: string
+                  envFrom:
```

Comment:
definition is [here](https://github.com/kubernetes/kubernetes/blob/7e2c02d4c86d73051b6d4f47298d35a7ffb42b6a/api/openapi-spec/v3/apis__apps__v1_openapi.json#L1681)

Reply:
Could you please add this as a comment?

Reply:
I fixed it with https://github.com/grafana/k6-operator/pull/129/commits/1559ba0f183fbc9c89fadd736f1dd17dc78bd354.